docs/sphinx/source/user_guide/weather_data.rst

```diff
@@ -0,0 +1,63 @@
+.. _weatherdata:
+
+Importing weather data
+======================
```

Comment:
```suggestion
Weather data
=========
```

Reply:
I think it's more than just "importing", it's also obtaining/retrieving weather data.